doc/source/security/encrypt-traffic.rst

```diff
@@ -12,19 +12,67 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable that
+enables automatic configuration of SSL encryption.
+
```

Comment:
Maybe we can make this clearer by making this into a numbered list of things to do
(1) create certs
(2) enable pxc-encrypt-cluster-traffic

Reply:
Nice idea!